spec/factories/conferences.rb

```diff
@@ -54,5 +54,41 @@
         conference.save!
       end
     end
+
+    factory :semi_conference do
```

Comment:
all those names are confusing, and you are duplicating code in the factory, why don't you just use the existing full_conference factory and simply make the splashpage **not** be public in your test?